lib/shopify_cli/messages/messages.rb

```diff
@@ -15,6 +15,7 @@ module Messages
       },
       core: {
         errors: {
+          missing_node: "Node is necessary for this command and was not found in the environment.",
```

Comment:
@MeredithCastile I need your help with this one :)

Reply:
Maybe we can follow the same style as this other error: `node is required to create a rails project. Download at https://nodejs.org/en/download.`

Reply:
Good idea, @gonzaloriestra. So something like this:

`Node is required to continue. Install node here: https://nodejs.org/en/download.`

Reply:
@MeredithCastile thanks! I'll update the message in another PR, as this was already merged because it was an urgent fix.

Reply:
Fixed here: https://github.com/Shopify/shopify-cli/pull/2055/commits/4ab4b393afab92658425afccc7552591915f0fce